### PR TITLE
set correct .travis.yml,invliad flag Error testing

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- let the ```PHP_CS_FIXER``` be the scriptcommand in ```composer.json```.
- set the ```php-coveralls``` in ```composer.json```.
- add the invalid flag Error testing.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,11 +26,15 @@
     "require-dev": {
         "amphp/phpunit-util": "^1.0",
         "friendsofphp/php-cs-fixer": "^2.9",
-        "phpunit/phpunit": "^6.5"
+        "phpunit/phpunit": "^6.5",
+        "php-coveralls/php-coveralls": "^1.0"
```
Comment (kelunik) [inline composer.json]: We use the `.phar` by design, because a developer never needs the tool locally and it might avoid possible future conflicts.

File: composer.json [hunk 2]
```diff
@@ -26,11 +26,15 @@
...
     },
     "config": {
         "platform": {
             "php": "7.0.0"
         }
+    },
+    "scripts": {
+        "fix-syntax": "php vendor/bin/php-cs-fixer --diff --dry-run -v fix"
```
Comment (kelunik) [inline composer.json]: If we want to add that, it should probably not use `--dry-run`. Does Composer allow adding arguments so that `composer fix-syntax` and `composer fix-syntax --dry-run` work? I haven't really used Composer scripts, yet.

File: test/FilterTest.php [hunk 3]
```diff
@@ -74,4 +74,16 @@ public function testExecutesAllTasksOnException() {
             $this->assertStringEqualsFile($files[2][1], '2');
         }
     }
+
+    public function testFilterWithInvalidFlag() {
```
Comment (kelunik) [inline test/FilterTest.php]: This doesn't test an invalid flag, it rather tests a `null` callback. Could you move that to a separate PR?

Comment (peter279k) [reply test/FilterTest.php]: All right, I set the error testing method.
How about the ```testFilterWithNullCallable``` method.

Comment (kelunik) [reply test/FilterTest.php]: Sounds fine. :+1:

File: composer.json (hunk 2 above)
Comment (peter279k) [reply composer.json]: It cannot. The Composer cannot add additional arguments.
And the ```composer fix-syntax --dry-run``` does not work.

**Comment by peter279k** [comment]
Because of my upcoming recruit training of Research and Development Substitute Services, I cannot review request changes immediately.

Please feel free to add the request changes :+1: .

File: composer.json (hunk 2 above)
Comment (peter279k) [reply composer.json]: Ok, it can add additional argument.
Just use ```composer fix-syntax -- --dry-run```.

**Comment by kelunik** [comment]
Thanks!
